### ISSUE TITLE
width

### ISSUE THREAD

**Issue by bbetter** [shamanland/facebook-like-button#3]
can i change the width of button on your widget? 'cause i 've just tried facebook native likeview and it seems to lack a possibility of changing button size in any way

**Comment by ok3141** [comment]
Of course, you can change attribute `android:layout_width` in a regular way.
